src/core/PageTemplate.js

```diff
@@ -0,0 +1,31 @@
+// Class Component
+import React from 'react'
+
+import { MainNav } from 'common/components/MainNav/index.js'
+import { Header } from 'common/components/Header/index.js'
+import { Aisles } from 'common/components/Aisles/index.js'
+
+// Function Component
+export const PageTemplate = ({ children }) => {
+
+  return (
+    <>
+      <MainNav/>
+      <Header/>
+      <Aisles/>
```

Comment:
These three components could be moved to another component, since they'll likely always be grouped.